torch/__init__.py

```diff
@@ -190,7 +190,7 @@ def _load_global_deps():
 
 # Appease the type checker; ordinarily this binding is inserted by the
 # torch._C module initialization code in C
-if False:
+if not _C:
```

Comment:
why this change?

Reply:
It appeases more type checkers but isn't necessary. I reverted this changed.